2.0.2/CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 # Changelog
 
-Astronomer Certified 2.0.2-1.dev, TBD
--------------------------------------
+Astronomer Certified 2.0.2-1, 2021-04-26
+----------------------------------------
 
-No User-facing CHANGELOG for AC 2.0.2+astro.1 from Airflow 2.0.2:
+User-facing CHANGELOG for AC 2.0.2+astro.1 from Airflow 2.0.2:
+
+### Bugfixes
+
+- Fix critical CeleryKubernetesExecutor bug (#13247) ([commit](https://github.com/astronomer/airflow/commit/02f780295))
+- Finish refactor of DAG resource name helper (#15511) ([commit](https://github.com/astronomer/airflow/commit/fd86869a8))
```

Comment:
This probably belongs under improvements.

Reply:
True, Done